### DISCUSSION TITLE
Expanded Metrics for Deliveries and Sales

### DISCUSSION THREAD

**Discussion by gnarlyvr1** [gameserverapp/Platform#620] [Suggestions]
Having expanded metrics would allow server owners to gauge what their player base is interested in and would allow them to tailor future shop items to that.

The ability to export to excel or a similar format would be beneficial as well for owners who want to manually look at their metrics.

**Comment by JetJeager** [comment]
Having the ability to export as CSV is instrumental!

**Comment by Decollates** [comment]
I am also very interested in seeing this implemented!

**Comment by DieselEffect** [comment]
Yes, please!

**Comment by JetJeager** [comment]
Max please take a look at this. a simple CSV of all item sales by shop pack.

**Comment by JetJeager** [comment]
This needs some love XD

**Comment by babsgosgens** [comment]
Great idea. I would love to be able to see separate revenue on single-purchase. Either a way to display that on the dashboard, or a way to export the data which will let me filter that outside of GSA would be nice.

**Comment by mvaessen** [comment]
What kind of info are you folks looking for? I'm looking for some examples of the kind of data / graphs you want to see etc.

**Reply by DieselEffect** [reply]
I'm extremely interested in the ability to export purchases to CSV so that I can sort through the data.

**Reply by mvaessen** [reply]
> I'm extremely interested in the ability to export purchases to CSV so that I can sort through the data.

make sure to upvote this item, if you like to see exports for sales data: #962

**Reply by babsgosgens** [reply]
First thing that came to kind, is most ordered items.

**Comment by JetJeager** [comment]
Hey max. I’d like to see x purchased and a breakdown by month.

Example.
5 purchases of x supporter pack
15 purchases of x shop pack

Being able to see a breakdown similar to how PayPal reports work.

**Comment by mvaessen** [comment]
For the folks interested: exporting sales data to a CSV is now available for all dashboards with an active add-on
